package.json

```diff
@@ -33,47 +33,48 @@
     "views"
   ],
   "dependencies": {
-    "acorn": "^5.3.0",
-    "bfj-node4": "^5.2.0",
-    "chalk": "^2.3.0",
-    "commander": "^2.13.0",
-    "ejs": "^2.5.7",
-    "express": "^4.16.2",
-    "filesize": "^3.5.11",
+    "acorn": "^5.5.3",
+    "bfj-node4": "^5.3.1",
```

Comment:
if it will drop node 4 support, maybe, it worth to use `bfj` package instead? It seems that the current one is deprecated...
https://www.npmjs.com/package/bfj-node4